addons/mail/static/tests/qunit_suite_tests/components/message_in_reply_to_tests.js

```diff
@@ -0,0 +1,113 @@
+/** @odoo-module **/
+
+import { start, startServer } from "@mail/../tests/helpers/test_utils";
+
+import { nextTick, patchWithCleanup } from "@web/../tests/helpers/utils";
+
+QUnit.module("mail", {}, function () {
+    QUnit.module("components", {}, function () {
+        QUnit.module("message_in_reply_to_tests");
+
+        QUnit.test(
+            "click on message in reply to highlight the parent message",
+            async function (assert) {
+                assert.expect(1);
+
+                const pyEnv = await startServer();
+                const mailChannelId1 = pyEnv["mail.channel"].create({});
+                const mailMessageId1 = pyEnv["mail.message"].create({
+                    body: "Hey lol",
+                    message_type: "comment",
+                    model: "mail.channel",
+                    res_id: mailChannelId1,
+                });
+                const mailMessageId2 = pyEnv["mail.message"].create({
+                    body: "Response to Hey lol",
+                    message_type: "comment",
+                    model: "mail.channel",
+                    parent_id: mailMessageId1,
+                    res_id: mailChannelId1,
+                });
+                const { click, openDiscuss } = await start({
+                    discuss: {
+                        params: {
+                            default_active_id: `mail.channel_${mailChannelId1}`,
+                        },
+                    },
+                });
+                await openDiscuss();
+                // TODO-DISCUSS-REFACTORING: remove when active id will be handle.
+                await click(".o-mail-category-item");
```

Comment:
Somehow this test does not pass.
```js
await openDiscuss(); // works
await click(".o-mail-category-item"); // not working (no re-render)
```

Same remark for the other test in this same file.